optbinning/binning/binning.py

```diff
@@ -523,7 +523,7 @@ def fit(self, x, y, sample_weight=None, check_input=False):
 
         Returns
         -------
-        self : object
```

Comment:
I do not disagree with this one, but to be consistent, all optbinning classes with a fit method should be changed.